Accessories/ExpansionBoard_uInverter/REV20220329B/uInv_revB_BOM.csv

```diff
@@ -0,0 +1,21 @@
+Supplier Part Number 1,Quantity,Comment,Description,Designator,Footprint,LibRef
+
+"399-17618-1-ND","3","0.1uF","Cap Ceramic 0.1uF 35V X7R 10% SMD 0805","C1A, C1B, C1C","0805_CAP","C0805C104K6RACTU"
+"490-10514-2-ND","5","10uF","0805 35 V 10 uF �10% Tolerance X5R Multilayer Ceramic Capacitor","C2A, C2B, C2C, C5, C6","0805_CAP","GRM21BR6YA106KE43L"
+"445-7551-1-ND","6","2.2uF","2.2 �F �10% 35V Ceramic Capacitor X7R 0805","C3A, C3B, C3C, C4A, C4B, C4C","0805_CAP","C2012X7R1V225K125AB"
```

Comment:
Can you remove the "funny" text characters from the BOM?

![image](https://user-images.githubusercontent.com/20168990/161800396-69959000-8871-41b6-afb9-65a79d97c24a.png)

Reply:
Removed.